pkg/signatures/cosign_sig_verifier.go

```diff
@@ -286,21 +360,26 @@ func cosignCheckOptsFromCert(cert certVerificationData) (cosign.CheckOpts, error
 		if err != nil {
 			return opts, err
 		}
+		opts.IntermediateCerts, err = fulcio.GetIntermediates()
```

Comment:
how about extracting it to a separated bugfix pr

Reply:
done